Rebuild-Black-Business/RBB-Website
# adds cta styling to justin's links in /about

# Describe your PR
Justins links in /about didnt have styling

![image](https://user-images.githubusercontent.com/8475305/84324955-b1aaa600-ab47-11ea-8ae5-ae8a8c9f6804.png)

Related to #
Fixes #

## Pages/Interfaces that will change
/about

### Screenshots / video of changes

Drop some screenshots of the before and after of your work here. Better yet, take a screen recording using a tool like [Loom](https://www.loom.com/)

![image](https://user-images.githubusercontent.com/8475305/84324847-685a5680-ab47-11ea-994a-464181fd3804.png)

## Steps to test

1. Go to /about
2. inspect Justins links
3. Should have styling for different states that follow similar to /ally card orange links

### Additional notes
